andela/bp-esa-backend
# #167297420: Remove the warning messages when running tests

#### What does this PR do?
- Clear error logs displayed when tests are run

#### Description of Task to be completed?
- Remove the error logs displayed when tests are run

#### How should this be manually tested?
1. Check out to the branch `bg-fix-failing-test-167297420`
2. On terminal, run command `yarn test`

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?


#### Postman Documentation
- N/A

#### Screenshots (If applicable)
<img width="1440" alt="Screen Shot 2019-07-29 at 4 37 30 PM" src="https://user-images.githubusercontent.com/9901086/62053099-9aa8e100-b21f-11e9-96ee-06bf2b7e63c9.png">
<img width="1440" alt="Screen Shot 2019-07-29 at 4 37 13 PM" src="https://user-images.githubusercontent.com/9901086/62053106-a09ec200-b21f-11e9-828c-225b5c631d3e.png">
